Cargo.toml

```diff
@@ -13,12 +13,11 @@ keywords = ["zk", "zookeeper", "distributed", "coordination"]
 [dependencies]
 byteorder = "0.4"
 bytes = "0.3"
-enum_primitive = "0.1"
 lazy_static = "0.1"
 log = "0.3"
 mio = "0.5"
-num = "0.1"
 snowflake = "1.0.0"
+zookeeper_derive = { path = "zookeeper-derive" }
```

Comment:
I added this so we can use `#[derive(EnumError, EnumConvertFromInt)]` instead of relying on hand-coding a match and `enum_primitive!`, respectively. Personally, I find this more readable and IDEA seems way better at parsing things.